server/sonar-web/src/main/js/apps/overview/meta/Meta.tsx

```diff
@@ -83,27 +77,28 @@ export default class Meta extends React.PureComponent<Props> {
           qualifier={component.qualifier}
         />
 
-        {shouldShowQualityGate && (
-          <MetaQualityGate
-            gate={qualityGate}
-            organization={hasOrganization && component.organization}
-          />
-        )}
-
-        {shouldShowQualityProfiles && (
-          <MetaQualityProfiles
-            component={component}
-            customOrganizations={organizationsEnabled}
-            organization={component.organization}
-            profiles={qualityProfiles}
-          />
-        )}
+        {isProject &&
+          qualityGate !== undefined && (
+            <MetaQualityGate
+              organization={organizationsEnabled ? component.organization : undefined}
+              qualityGate={qualityGate}
+            />
+          )}
+
+        {isProject &&
+          qualityProfiles !== undefined &&
+          qualityProfiles.length > 0 && (
+            <MetaQualityProfiles
+              organization={organizationsEnabled ? component.organization : undefined}
+              profiles={qualityProfiles}
+            />
+          )}
 
         {isProject && <MetaLinks component={component} />}
 
-        <MetaKey component={component} />
+        <MetaKey componentKey={component.key} qualifier={component.qualifier} />
 
-        {hasOrganization && <MetaOrganizationKey component={component} />}
+        {organizationsEnabled && <MetaOrganizationKey organization={component.organization} />}
```

Comment:
I think `MetaKey` and `MetaOrganizationKey` should also be grouped under the same `overview-meta-card`